2021-04-03 16:49:14,463 : ERROR: black pixel overflow
2021-04-03 16:49:14,506 : ERROR: deskew and binarize error
2021-04-03 16:49:14,524 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 16:49:14,524 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 16:54:05,633 : ERROR: black pixel overflow
2021-04-03 16:54:05,633 : ERROR: deskew and binarize error
2021-04-03 16:54:05,633 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 16:54:05,633 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 16:54:23,870 : ERROR: black pixel overflow
2021-04-03 16:54:23,870 : ERROR: deskew and binarize error
2021-04-03 16:54:23,870 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 16:54:23,870 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 16:54:35,337 : ERROR: black pixel overflow
2021-04-03 16:54:35,338 : ERROR: deskew and binarize error
2021-04-03 16:54:35,338 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 16:54:35,338 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 16:54:45,455 : ERROR: segment error
2021-04-03 16:54:46,116 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 96, in Slice
    segments = segment(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 58, in segment
    segmenter = Segmenter(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\segmenter.py", line 12, in __init__
    self.thickness, self.spacing = calculate_thickness_spacing(
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\staff.py", line 17, in calculate_thickness_spacing
    pair, count = Counter(flattened).most_common()[0]
IndexError: list index out of range

2021-04-03 16:54:46,117 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 96, in Slice
    segments = segment(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 58, in segment
    segmenter = Segmenter(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\segmenter.py", line 12, in __init__
    self.thickness, self.spacing = calculate_thickness_spacing(
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\staff.py", line 17, in calculate_thickness_spacing
    pair, count = Counter(flattened).most_common()[0]
IndexError: list index out of range

2021-04-03 16:55:27,058 : ERROR: segment error
2021-04-03 16:55:27,058 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 96, in Slice
    segments = segment(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 58, in segment
    segmenter = Segmenter(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\segmenter.py", line 12, in __init__
    self.thickness, self.spacing = calculate_thickness_spacing(
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\staff.py", line 17, in calculate_thickness_spacing
    pair, count = Counter(flattened).most_common()[0]
IndexError: list index out of range

2021-04-03 16:55:27,059 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 96, in Slice
    segments = segment(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 58, in segment
    segmenter = Segmenter(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\segmenter.py", line 12, in __init__
    self.thickness, self.spacing = calculate_thickness_spacing(
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\staff.py", line 17, in calculate_thickness_spacing
    pair, count = Counter(flattened).most_common()[0]
IndexError: list index out of range

2021-04-03 16:55:41,406 : ERROR: segment error
2021-04-03 16:55:41,407 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 96, in Slice
    segments = segment(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 58, in segment
    segmenter = Segmenter(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\segmenter.py", line 12, in __init__
    self.thickness, self.spacing = calculate_thickness_spacing(
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\staff.py", line 17, in calculate_thickness_spacing
    pair, count = Counter(flattened).most_common()[0]
IndexError: list index out of range

2021-04-03 16:55:41,407 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 96, in Slice
    segments = segment(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 58, in segment
    segmenter = Segmenter(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\segmenter.py", line 12, in __init__
    self.thickness, self.spacing = calculate_thickness_spacing(
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\staff.py", line 17, in calculate_thickness_spacing
    pair, count = Counter(flattened).most_common()[0]
IndexError: list index out of range

2021-04-03 16:56:56,898 : ERROR: segment error
2021-04-03 16:56:56,898 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 96, in Slice
    segments = segment(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 58, in segment
    segmenter = Segmenter(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\segmenter.py", line 12, in __init__
    self.thickness, self.spacing = calculate_thickness_spacing(
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\staff.py", line 17, in calculate_thickness_spacing
    pair, count = Counter(flattened).most_common()[0]
IndexError: list index out of range

2021-04-03 16:56:56,899 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 96, in Slice
    segments = segment(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 58, in segment
    segmenter = Segmenter(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\segmenter.py", line 12, in __init__
    self.thickness, self.spacing = calculate_thickness_spacing(
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\staff.py", line 17, in calculate_thickness_spacing
    pair, count = Counter(flattened).most_common()[0]
IndexError: list index out of range

2021-04-03 16:59:57,218 : ERROR: segment error
2021-04-03 16:59:57,219 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 96, in Slice
    segments = segment(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 58, in segment
    segmenter = Segmenter(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\segmenter.py", line 12, in __init__
    self.thickness, self.spacing = calculate_thickness_spacing(
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\staff.py", line 17, in calculate_thickness_spacing
    pair, count = Counter(flattened).most_common()[0]
IndexError: list index out of range

2021-04-03 16:59:57,219 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 96, in Slice
    segments = segment(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 58, in segment
    segmenter = Segmenter(imgmat)
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\segmenter.py", line 12, in __init__
    self.thickness, self.spacing = calculate_thickness_spacing(
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\staff.py", line 17, in calculate_thickness_spacing
    pair, count = Counter(flattened).most_common()[0]
IndexError: list index out of range

2021-04-03 17:25:40,851 : ERROR: black pixel overflow
2021-04-03 17:25:40,851 : ERROR: deskew and binarize error
2021-04-03 17:25:40,851 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 17:25:40,851 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 17:34:47,386 : ERROR: black pixel overflow
2021-04-03 17:34:47,386 : ERROR: deskew and binarize error
2021-04-03 17:34:47,386 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 17:34:47,386 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 17:59:59,275 : ERROR: black pixel overflow
2021-04-03 17:59:59,484 : ERROR: deskew and binarize error
2021-04-03 17:59:59,603 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 17:59:59,604 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 18:20:02,957 : ERROR: black pixel overflow
2021-04-03 18:20:02,958 : ERROR: deskew and binarize error
2021-04-03 18:20:03,059 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

2021-04-03 18:20:03,059 : Traceback (most recent call last):
  File "C:\Users\aroue\Downloads\Documents\@ML\notable-flask\segmenter\slicer.py", line 84, in Slice
    raise ValueError("black pixel overflow")
ValueError: black pixel overflow

